tests/test_queuing.py

```diff
@@ -406,22 +344,119 @@ def __init__(self, *args):
                 self.queue()
 
             def queue(self):
-                qml.QueuingContext.append(self, owns=tuple(self.obs))
+                QueuingContext.append(self, owns=tuple(self.obs))
 
                 for o in self.obs:
                     try:
-                        qml.QueuingContext.update_info(o, owner=self)
+                        QueuingContext.update_info(o, owner=self)
                     except AttributeError:
                         pass
 
                 return self
 
-        with qml._queuing.AnnotatedQueue() as q:
+        with AnnotatedQueue() as q:
             A = qml.PauliZ(0)
             B = qml.PauliY(1)
             tensor_op = AnnotatingTensor(A, B)
 
         assert q.queue == [A, B, tensor_op]
-        assert q.get_info(A) == {"owner": tensor_op}
-        assert q.get_info(B) == {"owner": tensor_op}
-        assert q.get_info(tensor_op) == {"owns": (A, B)}
+        assert q._get_info(A) == {"owner": tensor_op}
+        assert q._get_info(B) == {"owner": tensor_op}
+        assert q._get_info(tensor_op) == {"owns": (A, B)}
+
+
+class TestOperationRecorder:
+    """Test the OperationRecorder class."""
+
+    def test_circuit_integration(self):
+        """Tests that the OperationRecorder integrates well with the
+        core behaviour of PennyLane."""
```

Comment:
These docstrings seem to be identical for each test case, worth making them a bit more descriptive.